scripts/irods/test/test_all_rules.py

```diff
@@ -1119,6 +1119,55 @@ def do_test(logical_path, acl):
             do_test(data_object, 'own')
             self.admin.assert_icommand(['imeta', 'ls', '-d', data_object], 'STDOUT', ['atomic_pre_fired', 'atomic_post_fired'])
 
+    @unittest.skipIf(database_instance_name == 'mysql',
+                    ("Fails against databases with transaction isolation level set to REPEATABLE-READ (e.g. MySQL). "
+                     "For more details, see https://github.com/irods/irods/issues/4917"))
+    def test_msi_atomic_apply_acl_operations_considers_group_permissions__issue_6191(self):
```

Comment:
Same here. Does this need to be skipped if the REP instance is not `irods_rule_engine_plugin-irods_rule_language`?

Reply:
definitely

Reply:
added skip condition